src/modules/utils/motordrivercontrol/MotorDriverControl.cpp

```diff
@@ -54,6 +63,21 @@ MotorDriverControl::~MotorDriverControl()
 // this will load all motor driver controls defined in config, called from main
 void MotorDriverControl::on_module_loaded()
 {
+    // Enabled?
+    if (!THEKERNEL->config->value(motor_driver_control_checksum, enable_checksum )->by_default(1)->as_bool()) {
+        delete this;
+        return;
+    }
+
+    // Wait for drivers
+    int timeout = THEKERNEL->config->value(motor_driver_control_checksum, timeout_checksum)->by_default(0)->as_number();
+
+    if (timeout > 0)
+    {
+        wait(timeout);
```

Comment:
If wait does what I think it does it is not allowed. it is not allowed to halt processing for any reason at anytime.

Reply:
It's exaclty as it seems, however it's a heritage from the previous versions; I'm not sure if it's really needed, but since it default's to 0 and used only at init time I don't think it's a big issue to have it here. Anyway, if you say so, I'l remove it.

Reply:
yes it needs to be removed. wait() is NOT a valid call that is allowed anywhere in smoothie. I must have missed it in the other requests.

Reply:
Removed as requested.